tests/lock_test.py

```diff
@@ -12,20 +12,19 @@
 import sys
 import tempfile
 
-from cryptography import x509
-from cryptography.hazmat.backends import default_backend
 # TODO: once mypy has cryptography types bundled, type: ignore can be removed.
```

Comment:
another comment that needs to stay with the thing it's talking about

Reply:
Fixed that.